server/eb/README.md

```diff
@@ -0,0 +1,65 @@
+# AWS Elastic Beanstalk 
```

Comment:
Please add a large, bold caveat to the effect that this is an experimental, incomplete feature, subject to change.  And that we would love feedback :-)